File: QA Pre-Academy Learning with Assessment/QA Post Pre-Learning task/calculator.py
def calculator(a, b, operator):
    # ==============
    # Your code here
    if operator == '+':
        result = a + b
    elif operator == '-':
        result = a - b
    elif operator == '*':
        result = a * b
    else:
        result = a / b
    print(int(result))
    # ==============

print(calculator(2, 4, "+")) # Should print 6 to the console
print(calculator(10, 3, "-")) # Should print 7 to the console
print(calculator(4, 7, "*")) # Should print 28 to the console
print(calculator(100, 2, "/")) # Should print 50 to the console

File: Training Notes/Java Beginner/27.05.22/QAJavaWorkspace/NewCalculator/src/com/newcalculator/methods/Calculator.java
package com.newcalculator.methods;

import java.util.Scanner;

public class Calculator {
	// Attributes
//	private int num1;
//	private int num2;
//	private String operator;

	// Default Constructor
	public Calculator() {
	}

	// Everything Constructor
//	public Calculator(int num1, int num2, String operator) {
//		this.num1 = num1;
//		this.num2 = num2;
//		this.operator = operator;
//	}
	Scanner calc = new Scanner(System.in);
	
	public void runCalc() {

		System.out.println("First number: ");
		int num1 = calc.nextInt();

		System.out.println("Second number: ");
		int num2 = calc.nextInt();

		calc.nextLine();

		System.out.println("operator of choice (+, -, /, *)");
		String operator = calc.next();

		switch (operator) {
		case "+":
			System.out.println("Result: " + (num1 + num2));
			break;
		case "-":
			System.out.println("Result: " + (num1 - num2));
			break;
		case "*":
			System.out.println("Result: " + (num1 * num2));
			break;
		case "/":
			System.out.println("Result: " + (num1 / num2));
		}
	}

}
